clang/lib/StaticAnalyzer/Core/BugReporterVisitors.cpp

```diff
@@ -1634,14 +1634,15 @@ PathDiagnosticPieceRef StoreSiteFinder::VisitNode(const ExplodedNode *Succ,
           ProgramStateManager &StateMgr = BRC.getStateManager();
           CallEventManager &CallMgr = StateMgr.getCallEventManager();
 
-          CallEventRef<> Call = CallMgr.getCaller(CE->getCalleeContext(),
```

Comment:
@ziqingluo-90 It looks like there's a lot of whitespace-only changes. Would you mind trying to minimize the patch so it contains only the relevant changes?

It looks like the diff is significantly smaller in my original PR:
https://github.com/apple/llvm-project/pull/5352/files
Is it possible that some of these changes were introduced by `clang-format`?

Reply:
Thanks for pointing this out.  I didn't notice that`clang-format` was automatically applied. I have now updated the [commit](https://github.com/apple/llvm-project/pull/5362/commits/2fafab401febebafe1c5bd3f2ac161ae34ec68b6).